scripts/ansible/roles/linux/az-dcap-client/tasks/ubuntu/stable-install.yml

```diff
@@ -21,3 +21,31 @@
     update_cache: yes
   # az-dcap-client is not available for Ubuntu 22.04
   when: ansible_distribution_release != 'jammy'
+
+- name: Download libssl1.1 Debian package
```

Comment:
This task, and the installation of the .deb doesn't belong in the az-dcap-client role. This should go into the OE role as part of environment setup. 

Also this, and the other download task should only run on Jammy.

Reply:
The Az DCAP client tasks should stay in the az-dcap-client role